assets/robotest/resources/install.yaml

```diff
@@ -0,0 +1,22 @@
+ apiVersion: batch/v1
+ kind: Job
+ metadata:
+   name: echoserver-install
+   namespace: default
+ spec:
+   template:
+     metadata:
+       name: echoserver-install
+     spec:
+       restartPolicy: Never
+       containers:
+         - name: echoserver-install
+           image: quay.io/gravitational/debian-tall:buster
+           command:
+             - /usr/local/bin/helm
```

Comment:
I would also set a release name (`--name xxx`) so it can be used in "helm upgrade" command in the upgrade hook.